ethicml/algorithms/inprocess/shared.py

```diff
@@ -26,17 +28,17 @@ def flag_interface(in_algo_args: InAlgoArgs, flags: Mapping[str, Any]) -> List[s
 
 
 def settings_for_svm_lr(
-    classifier: str, C: Optional[float], kernel: Optional[str]
-) -> Tuple[float, str]:
+    classifier: ClassifierType, C: Optional[float], kernel: Optional[KernelType]
+) -> Tuple[float, Optional[KernelType]]:
     """If necessary get the default settings for the C and kernel parameter of SVM and LR."""
     if C is None:
-        if classifier == "LR":
+        if classifier == ClassifierType.lr:
             C = LogisticRegression().C
-        elif classifier == "SVM":
+        elif classifier == ClassifierType.svm:
             C = SVC().C
         else:
             raise NotImplementedError(f'Unsupported classifier "{classifier}".')
 
     if kernel is None:
-        kernel = SVC().kernel if classifier == "SVM" else ""
+        kernel = KernelType[SVC().kernel] if classifier == ClassifierType.svm else None
```

Comment:
```suggestion
        kernel = KernelType[SVC().kernel] if classifier is ClassifierType.svm else None
```

Reply:
I do search for these... not sure how they keep getting missed